types/recharts/recharts-tests.tsx

```diff
@@ -136,12 +136,8 @@ class Component extends React.Component<{}, ComponentState> {
                 </ResponsiveContainer>
                 <ResponsiveContainer>
                     <LineChart width={500} height={300} data={data}>
-                        <XAxis dataKey="name">
-                            <Label>X axis - name</Label>
```

Comment:
Why do you remove both `Label` elements? I thought it should be possible to use both.